packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/java/integrationsteps.md

```diff
@@ -1,3 +1,272 @@
+
+<div class="common-image-format">
+
+![Enroll phone form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-auth-list-phone-java.png
+ "Enroll phone authenticator form for Java SDK")
+
+</div>
+
+### Step 8, Option 1: User selects phone authenticator
+
+In this use case option, the user selects the optional **Phone** factor as the authenticator to verify. Pass this selected authenticator to the `IDXAuthenticationWrapper.selectAuthenticator` method.
+
+```java
+authenticationResponse = idxAuthenticationWrapper.selectAuthenticator(proceedContext, authenticator);
+```
+
+The response from this request is an [AuthenticationResponse](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java) object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_DATA`. This status indicates that the user needs to provide additional authenticator information. In the case of the phone authenticator, the user needs to specify a phone number, and whether they want to use SMS or voice as the verification method.
+
+### Step 9, Option 1: User enters phone number and selects phone verify method
+
+You need to build a form to capture the user's phone number as well as a subsequent form for the user to select their phone verification method (either SMS or voice).
+
+<div class="common-image-format">
+
+![Enroll phone number for Java SDK](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-num-java.png
+ "Displays the enroll phone number authenticator form for Java SDK")
+
+</div>
+
+> **Note:** The Java SDK requires the following phone number format: `{+}{country-code}{area-code}{number}`. For example, `+15556667777`.
+
+<div class="common-image-format">
+
+![Enroll phone method](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-mode-java.png
+ "Displays the enroll phone method (SMS or voice) form for Java SDK")
+
+</div>
+
+> **Note:** Only SMS is currently supported as the phone authenticator method for the Java SDK.
+
+When the user enters their phone number and selects SMS to receive the verification code, capture this information and send it to [IDXAuthenticationWrapper](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java)'s `submitPhoneAuthenticator` method.
+
+For example, the following code snippet passes the `phone` and `mode` variables to `idxAuthenticationWrapper.submitPhoneAuthenticator`:
+
+```java
+if (!Strings.hasText(phone)) {
```

Comment:
Remove lines 219 to 229 from the code snippet. Need not show code that deals with views.